lib/lapis_lazuli/generators/cucumber/template/features/support/env.rb

```diff
@@ -5,7 +5,8 @@
 require 'lapis_lazuli'
 require 'lapis_lazuli/cucumber'
 
-LapisLazuli::WorldModule::Config.config_file = "config/config.yml"
+LapisLazuli::WorldModule::Config.add_config("config/config.yml")
+LapisLazuli::WorldModule::Config.add_config("config/users.yml")
```

Comment:
What happens when you have similar items in the config files? Will that overwrite the initially created value f a previous config file?

Reply:
The yml file is converted to a `hash` and then `.merge!` into `@config`.  So yes, similar items in the config are overwritten by the last file added. This also allows for things like:
```
add_config('default-config.yml')
if ENV['test_env'] == 'staging'
  add_config('staging-config.yml')
end
``` 
`staging-config.yml` will then overwrite any duplicate values defined in config.